style.css

```diff
@@ -0,0 +1,162 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    font-family: Annlie
+}
+ul {
+    list-style: none;
+}
+.container {
+    display: grid;
+    grid-template: 
+        "left-bar main right-bar" 1fr /
+        1fr 700px 1fr;
+    margin-left: 400px;
```

Comment:
I noticed that I can't see that well the right column because I wasn't using at the full screen the windows of my browser. To help with that (or users with a small screen) I thought about using an auto margin an putting  the container at the  center of the page